source/slang/slang-compiler.h

```diff
@@ -2815,6 +2818,7 @@ namespace Slang
         virtual SLANG_NO_THROW void SLANG_MCALL setTargetForceDXLayout(int targetIndex, bool value) SLANG_OVERRIDE;
         virtual SLANG_NO_THROW void SLANG_MCALL setTargetGenerateWholeProgram(int targetIndex, bool value) SLANG_OVERRIDE;
         virtual SLANG_NO_THROW void SLANG_MCALL setEmbedDXIL(bool value) SLANG_OVERRIDE;
+        virtual SLANG_NO_THROW void SLANG_MCALL setEmbedSPIRV(bool value) SLANG_OVERRIDE;
```

Comment:
Can we just use one single option called `embedDownstreamIR`?

Reply:
Yes

Reply:
Done